episodes/11-exercises.md

```diff
@@ -114,33 +142,95 @@ and that assesses their ability to do so.
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
-Well-designed exercises are one of the most valuable resources for an Instructor:
-time spent on this is well invested.
+## Demo of Writing an Exercise 
+
+Well-designed exercises are one of the most valuable resources for an instructor and 
+any time spent on this is well invested.
+
+<!--- will there be an example lesson that the trainer is working on ? -->
+
+To create an exercise in the Carpentries Workbench, 
+you can use colon-delimited sections called 'fenced divs'.
+In fact there are many types of boxes in the lesson infrastucture, called "special blockquotes", 
+that use fenced divs.
+To start a challenge fenced div the line must contain at least 3 colons, then the `challenge` tag.
```

Comment:
At the moment, there's no plan to add "exercise" as a valid type, especially if it can refer to two valid types (that being said, having "exercise" be a generic type for "discussion" if there is a solution block and "challenge" if there is no solution block is an interesting concept to explore in the future)

Reply:
At the moment, [the fenced-div validator](https://carpentries.github.io/pegboard/articles/validation.html#fenced-div-validation) will catch those and throw the error `file:#xx  [unknown div] exercise`, but I think I can add extra context for that particular one to suggest "did you mean challenge or discussion?"